### PR TITLE
Issue 1753 avatar nav link

### LINKED ISSUE #1729: Avatar button labels aren't too friendly w/ l10n strings
Found in https://accounts.stage.mozaws.net/ver.json (commit 7ff9e196311cef2e9e5a1726aa49081f2c0167a5)

```json
{
  "version": "0.22.0",
  "commit": "7ff9e196311cef2e9e5a1726aa49081f2c0167a5",
  "l10n": "9c6f728ea6",
  "tosPp": "2becbeea57"
}
```

---

### Steps to reproduce:
1. Log in and go to https://accounts.stage.mozaws.net/settings/avatar/change using a non-_en-US_ locale. I randomly happened to be in "de German".

### Actual results:
Looks like the "n" in "Hochladen" is escaping the circle of doom.
kontobild

### Expected results:
Non rounded buttons, so the buttons can size to fit locales w/ awkwardly long words.

---

preferences

### LINKED ISSUE #1753: avatar "manage account" link is inconsistent with other navigation links
On `/settings/avatar`, the "manage account" link is above the header. In all other pages, our navigation links are at the bottom. I found this a bit confusing.

screen shot 2014-10-08 at 10 48 55

### PR DESCRIPTION
**Pull request by zaach**
The avatar change screen is looking fresh now. This makes the nav l10n friendly and removes that weird subheader nav from avatar pages.

screen shot 2015-01-08 at 12 14 44 am

Fixes #1753.
Fixes #1729.

### CODE REVIEW

**Comment by zaach** [comment]
@shane-tomlinson r?

**Comment by zaach** [comment]
This also fixes #1730.

File: app/scripts/templates/settings/avatar_change.mustache [hunk 1]
```diff
@@ -27,5 +26,9 @@
       <a href="/settings/avatar/camera" id="camera"><span class="icon-camera-alt"></span>{{#t}}Camera{{/t}}</a>
       <a href="/settings/avatar/gravatar" id="gravatar"><span class="gravatar-logo"></span>{{#t}}Gravatar{{/t}}</a>
     </nav>
+
+    <p id="settings-home">
+      <a href="/settings">{{#t}}Home{{/t}}</a>
```
Comment (pdehaan) [inline app/scripts/templates/settings/avatar_change.mustache]: "Home" or "Settings"?

File: app/styles/_avatars.scss [hunk 2]
```diff
@@ -63,13 +63,13 @@ a.avatar-wrapper {
   color: transparent;
 }
 
-#fxa-settings-sub-header {
-  font-size: $medium-font;
-  font-weight: normal;
-  padding-bottom: 5px;
+#settings-home {
+  margin: 40px 0px -30px;
```
Comment (pdehaan) [inline app/styles/_avatars.scss]: You can drop the "px" units on 0px.
Not sure if you want to sort these A-Z (same with the new rules below).

Comment (shane-tomlinson) [inline app/styles/_avatars.scss]: +1 on A-Z!

File: app/scripts/templates/settings/avatar_change.mustache (hunk 1 above)
Comment (zaach) [inline app/scripts/templates/settings/avatar_change.mustache]: Ryan's mocks say "Home" in #1753 but "Settings" would not be wrong.

**Comment by shane-tomlinson** [comment]
Review looks good, more testing needed. I'm re-installing my fxa-local-dev b/c images aren't processing for some reason. graphicsmagic is installed.

**Pull request merged**